### ISSUE TITLE
More tools need to be defined

### ISSUE THREAD

**Issue by frazik-main** [frazik-main/InstructGPT#3]
Let's define which tools we are going to implement.
We need tools like running code,  running new agents, etc.

**Comment by frazik-main** [comment]
@MatKollar, what do you think? Which tools do we need. Maybe filesystem tool, that can write to files?

**Comment by MatKollar** [comment]
Yes, a filesystem would be cool, with which it would be possible to read, write, append or delete files. 
Then maybe a tool to access the shell and execute commands.

**Comment by frazik-main** [comment]
We had discussion that tools like OCR models and Opean AI Clip can be tools.
